### ISSUE TITLE
Client seems to be very in favour of Windows clients

### ISSUE THREAD

**Issue by robmadeyou** [7winds/Project-Aeolus-Official-Client#3]
I know *nix clients aren't a majority, but still shouldn't restrict to only people on Windows. For example, it's impossible to log in without having windows, as an Exception is  thrown saying can't connect to the server.

I've got a fix for this, but it's very basic. Thought to let you know first before creating another pull request

**Comment by 7winds** [comment]
Yeah I don't have any other operating system to perform tests on. Post all the problems you have and I'll see what I can do.

**Comment by robmadeyou** [comment]
Would you like me to try and post fixes / updates for these? I'm looking to start up a server of my own and thought this would be a nice base; but I primarily run Linux so looking to get these fixes in as soon as possible.
